saleor/graphql/checkout/types.py

```diff
@@ -109,6 +110,7 @@ class Checkout(CountableDjangoObjectType):
         TaxedMoney,
         description="The price of the checkout before shipping, with taxes included.",
     )
+    token = graphene.Field(UUID, description=("UUID of checkout."), required=True)
```

Comment:
I would change it to have consistency with other description of that field above:
```suggestion
    token = graphene.Field(UUID, description=("The checkout's token."), required=True)
```

Reply:
Done